src/Messages/Attachments/OWSAttachmentsProcessor.m

```diff
@@ -79,7 +79,10 @@ - (instancetype)initWithAttachmentProtos:(NSArray<OWSSignalServiceProtosAttachme
 
         [attachmentIds addObject:pointer.uniqueId];
 
-        if ([MIMETypeUtil isSupportedMIMEType:pointer.contentType]) {
+        // TODO: We need to revisit isSupportedMIMEType.
+        if ([MIMETypeUtil isSupportedMIMEType:pointer.contentType] ||
+            [pointer.contentType isEqualToString:OWSMimeTypeOversizeTextMessage] ||
+            [pointer.contentType isEqualToString:OWSMimeTypeUnknownForTests]) {
```

Comment:
When we want to allow users to send arbitrary outgoing attachments as well, we should rework `isSupportedMIMEType`.